### PR TITLE
Status update for 2021-03-05

### PR DESCRIPTION
**Pull request by sheenaajay**

### CODE REVIEW

File: _posts/2021-03-05-status-update.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## Supported Module Major Releases
+
+The whole team has been super busy this week working on prepping our modules for major releases.
+We have done **34+** major releases on our supported modules with only a few outstanding.
+The release includes the removal of Puppet 5 support.
+If you are running Puppet 5 please note it is no longer supported and we advise you to upgrade to Puppet 7 as soon as you can.
+
```
Comment (david22swan) [inline _posts/2021-03-05-status-update.md:41]: Maybe also mention the removal of RHEL family 5 and SLES 11 support?

Comment (sheenaajay) [reply _posts/2021-03-05-status-update.md:41]: Perfect @david22swan . Thank you

File: _posts/2021-03-05-status-update.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+## Cloud CI
+
+This week we had the first test use of the automated release workflow and we found a couple of issues that we are currently working on (the pdk nightly image).
+We are also preparing a workflow to do the tag and push to forge. Link to the [PR][pdk-templates-pr-408].
+K8s module [PR](https://github.com/puppetlabs/puppetlabs-kubernetes/pull/481) is now merged which ports the module to cloud ci.
+
+
+## Update from our intern
+
+[Disha] completed her first community reporting PR which excludes the draft PR from module report generated for the community day.
```
Comment (sanfrancrisko) [inline _posts/2021-03-05-status-update.md:95]:
```suggestion
[Disha][Disha] completed her first community reporting PR which excludes the draft PR from module report generated for the community day.
```

**Review by sanfrancrisko — COMMENTED** [review]
One minor issue, but otherwise LGTM!

File: _posts/2021-03-05-status-update.md (hunk 2 above)
Comment (sheenaajay) [reply _posts/2021-03-05-status-update.md]: @sanfrancrisko Thank you...

**Review by david22swan — APPROVED** [review]

**Pull request merged**